app/models/occupation_standard.rb

```diff
@@ -23,9 +23,9 @@ class OccupationStandard < ApplicationRecord
   delegate :name, to: :industry, prefix: true, allow_nil: true
   delegate :state, to: :registration_agency, allow_nil: true
 
-  enum ojt_type: [:time, :competency, :hybrid], _suffix: :based
-  enum national_standard_type: [:program_standard, :guideline_standard, :occupational_framework], _prefix: :national
-  enum status: [:importing, :in_review, :published]
+  enum :ojt_type, [:time, :competency, :hybrid], :suffix => :based
```

Comment:
Same here. We could use `suffix: :based` and `prefix: :national`. You can see more examples in the [ActiveRecord::Enum documentation for Rails 8.01](https://api.rubyonrails.org/classes/ActiveRecord/Enum.html). But again, maybe there's a reason we are not using it. ☺️